core/client/fs/src/main/java/alluxio/client/file/cache/store/LocalPageStore.java

```diff
@@ -71,7 +72,7 @@ public void put(PageId pageId,
       }
       // extra try to ensure output stream is closed
       try (FileOutputStream fos = new FileOutputStream(pagePath.toFile(), false)) {
-        fos.write(page);
+        fos.getChannel().write(page);
```

Comment:
I'm curious what's the difference in terms of performance between the two interfaces, output stream and file channel?

Reply:
as we discussed offline, there is no difference for the write performance here..  I changed the put interface from byte to nio byte buffer just because I wanna take advantage of the nio buffer pool.